vscode/package.json

```diff
@@ -497,6 +497,10 @@
               "description": "Opt-in/out of the Tapioca add-on",
               "type": "boolean"
             },
+            "newTestExperience": {
+              "description": "UNDER DEVEOPMENT. Opt-in/out of the new test experience",
+              "type": "boolean"
```

Comment:
Does `all` cover this?


```json
    "rubyLsp.featureFlags": {
        "all": true,
    },
```

Reply:
It doesn't after #3119 goes in. This branch depends on that one.